packages/web/src/app/my-page/travel-log/page.tsx

```
@@ -1,3 +1,4 @@
+import { useModalContext } from "@/service/modal";
```

ESLint

[error] 'useModalContext' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)